app/scripts/metamask-controller.js

```diff
@@ -1291,6 +1291,21 @@ export default class MetamaskController extends EventEmitter {
       },
     );
 
+    this.controllerMessenger.subscribe(
+      `${this.snapController.name}:snapUpdated`,
+      (newSnap, oldVersion) => {
+        this.metaMetricsController.trackEvent({
+          event: 'Snap Updated',
+          category: EVENT.CATEGORIES.SNAPS,
+          properties: {
+            snap_id: newSnap.id,
+            oldVersion,
+            newVersion: newSnap.version,
```

Comment:
All properties must be snake cased. Is this event tracked in John Brennan's schema?